fe/fe-core/src/test/java/com/starrocks/analysis/ShowCreateViewStmtTest.java

```diff
@@ -0,0 +1,87 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.analysis;
+
+import com.clearspring.analytics.util.Lists;
+import com.starrocks.catalog.Catalog;
+import com.starrocks.catalog.Table;
+import com.starrocks.common.Config;
+import com.starrocks.common.FeConstants;
+import com.starrocks.qe.ConnectContext;
+import com.starrocks.utframe.StarRocksAssert;
+import com.starrocks.utframe.UtFrameUtils;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.UUID;
+
+public class ShowCreateViewStmtTest {
+
+    private static String runningDir = "fe/mocked/ShowCreateViewStmtTest/" + UUID.randomUUID().toString() + "/";
+
+    private static ConnectContext connectContext;
+    private static StarRocksAssert starRocksAssert;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        FeConstants.runningUnitTest = true;
+        FeConstants.default_scheduler_interval_millisecond = 100;
+        Config.dynamic_partition_enable = true;
+        Config.dynamic_partition_check_interval_seconds = 1;
```

Comment:
Is it necessary for this ut?

Reply:
it is not necessary, these code could be remove.